app/javascript/components/users/SignInComponent.tsx

```diff
@@ -57,47 +78,186 @@ function SignInComponent(props:SignInComponentProps) : JSX.Element {
 		}
 	}, [isValid, componentState]);
 
+
+	//Error messages
 	const { formatMessage } = useIntl();
+	const label = formatMessage({ id: 'email', defaultMessage: '* Requiered' });
+
+
+	//Yup validation
+	const validationSchema = yup.object({
+		email: yup.string().email().required(),
+		password: yup.string().required(),
+	});
+
+	//Styling
+	const useStyles = makeStyles((theme: Theme) => createStyles({
+		textField: {
+			'& .MuiTextField-root': {
+				margin: theme.spacing(1),
+			},
+		},
+		paper: {
+			maxWidth: 400,
+			margin: `${theme.spacing(1)}px auto`,
+			padding: theme.spacing(2),
+			borderRadius: 15,
```

Comment:
Does this border radius look good on all different display sizes?

Reply:
We've tested with different devices and it looks good

Reply:
Good! We should discuss the best way to do so.